### DISCUSSION TITLE
Error: Couldn't find pods targets.

### DISCUSSION THREAD

**Discussion by faimin** [swiftyfinch/Rugby#82] [Q&A] [answered]
```shell
> rugby 
Prepare Calculate checksums ⏱
⛔️ Error: Couldn't find pods targets.
🚑 Try to call pod install.
🚑 Call doctor for more help: rugby doctor
```

rugby version: `1.13.0`
Xcode: `13.1`
Mac OS: `11.6.1`

test demo： [https://github.com/faimin/ZDFlexLayoutKit](https://github.com/faimin/ZDFlexLayoutKit)  ,  branch: `dev`

**Comment by EkkoG** [comment, marked as answer]
run `pod install` before run `rugby`

**Reply by swiftyfinch** [reply]
@faimin @cielpy Hello!
@faimin Could you try the solution from @cielpy? Mark the answer if it helps you ✅ 

You can find more information [here](https://github.com/swiftyfinch/Rugby#how-to-use-):

> Run in your project directory after each pod install:
```shell
pod install && rugby
```

**Reply by faimin** [reply]
@cielpy  @swiftyfinch   I've tried `pod install && rugby` before the issue open.

![image](https://user-images.githubusercontent.com/8207436/142420731-e68acc93-9356-4af9-a809-66aece5a311d.png)

**Reply by swiftyfinch** [reply]
@faimin Oh, I see. I will try to check it in debug mode soon.

**Reply by faimin** [reply]
ok, thanks.

**Reply by faimin** [reply]
I found the problem. 
`rugby` not support `:generate_multiple_pod_projects` setting. 
when i close the setting, it work.

-----

my demo setting in `podfile`:

```ruby
install! 'cocoapods',
         :generate_multiple_pod_projects => true,
         :incremental_installation => true,
         :disable_input_output_paths => true,
         :preserve_pod_file_structure => true
```

**Reply by faimin** [reply]
@swiftyfinch  Is there any plan to support multiple pod projects ?

**Reply by swiftyfinch** [reply]
@faimin Yeah, thank you for the investigation!
🏈 Rugby operates with `Pods.xcodeproj`, creates a target into it, and builds it.

So, if we have a lot of separate projects we should build them one by one.
But it's not optimal.

Or we can create another project and add it all to it.
This seems like a complex feature.

No, I didn't plan to add it. Are you sure that this one is important for you?

**Reply by swiftyfinch** [reply]
## Please, feel free to open new issue with these templates:

<img width="407" alt="Screenshot 2021-11-21 at 21 01 19" src="https://user-images.githubusercontent.com/64660122/142773605-3715f9cb-e4d8-4c8b-8ed2-fc2ce9668263.png">

**Reply by faimin** [reply]
OK, thanks, i see. Mainly because our project has multiple projects on, and our pod hook settings base on multiple projects too. If we disable multiple projects, there will be a lot of changes to do.
